packages/tailwindcss-language-server/src/testing/index.ts

```diff
@@ -1,13 +1,16 @@
-import { onTestFinished, test, TestOptions } from 'vitest'
+import { onTestFinished, test, TestContext, TestOptions } from 'vitest'
 import * as os from 'node:os'
 import * as fs from 'node:fs/promises'
 import * as path from 'node:path'
 import * as proc from 'node:child_process'
 import dedent from 'dedent'
 
-export interface TestUtils {
+export interface TestUtils<TestInput extends Record<string, any>> {
   /** The "cwd" for this test */
   root: string
+
+  /** The "cwd" for this test */
```

Comment:
copy-paste error? I think `root` is already the `"cwd"` for this test

Reply:
I swore I fixed this lol thanks will update